pkgs/development/python-modules/manim/default.nix

```diff
@@ -257,25 +253,29 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     ffmpeg
     manim-tinytex
+    pytest-cov-stub
     pytest-xdist
     pytestCheckHook
+    versionCheckHook
```

Comment:
`versionCheckHook` should be in `nativeInstallCheckInputs`, and `doInstallCheck` should be set.

Reply:
Let's ask @mweinelt

Reply:
For Python installPhase is mapped to the installCheckPhase :man_shrugging: 

https://github.com/NixOS/nixpkgs/pull/324124